src/components/Dashboard/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './dashboard.scss';
+import Card from './DashboardCard';
+
+class Dashboard extends React.Component {
+  // Use case Example for dashboard card
```

Comment:
I think it would be better to use multi-line comments instead of single-line comments.